DXRMapFixups/DeusEx/Classes/DXRFixupVandenberg.uc

```diff
@@ -637,6 +637,10 @@ function AnyEntryMapFixes()
 
     case "14_VANDENBERG_SUB":
         FixSavageSkillPointsDupe();
+
+        GetConversation('JockArea51').AddFlagRef('dummy', True); // 'JockArea51' can never play
+        GetConversation('JockBarks').AddFlagRef('DL_Dead_Played', dxr.flagbase.GetBool('DL_Dead_Played')); // 'JockBarks' can always play
```

Comment:
I think this makes the intent a little more obvious, and it's a little cleaner

```suggestion
        DeleteConversationFlag(GetConversation('JockBarks'), 'DL_Dead_Played', false); // 'JockBarks' can always play
```

Reply:
I agree. I probably would have used that function if I knew about it.